ServiceWebsite/ServiceWebsite/Controllers/HealthCheckController.cs

```diff
@@ -0,0 +1,97 @@
+﻿using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using ServiceWebsite.BookingsAPI.Client;
+using ServiceWebsite.Models;
+using ServiceWebsite.UserAPI.Client;
+using Swashbuckle.AspNetCore.Annotations;
+using System;
+using System.Net;
+using System.Reflection;
+using System.Threading.Tasks;
+
+namespace ServiceWebsite.Controllers
+{
+    [Produces("application/json")]
+    [Route("HealthCheck")]
+    [AllowAnonymous]
+    [ApiController]
+    public class HealthCheckController : Controller
+    {
+        private readonly IUserApiClient _userApiClient;
+        private readonly IBookingsApiClient _bookingsApiClient;
+
+        public HealthCheckController(IUserApiClient userApiClient, IBookingsApiClient bookingsApiClient)
+        {
+            _userApiClient = userApiClient;
+            _bookingsApiClient = bookingsApiClient;
+        }
+
+        /// <summary>
+        /// Check Service Health
+        /// </summary>
+        /// <returns>Error if fails, otherwise OK status</returns>
+        [HttpGet("health")]
+        [SwaggerOperation(OperationId = "CheckServiceHealth")]
+        [ProducesResponseType(typeof(HealthCheckResponse), (int)HttpStatusCode.OK)]
+        [ProducesResponseType(typeof(HealthCheckResponse), (int)HttpStatusCode.InternalServerError)]
+        public async Task<IActionResult> Health()
+        {
+            var response = new Models.HealthCheckResponse
+            {
```

Comment:
Hi Lionel, Could you add the Kinly API check here as well? You can see the implementation in VideoWeb

Reply:
Added the check for Video API